README.md

```diff
@@ -6,76 +6,76 @@
+Starting from Kodi Leia (v18), you can install Retrospect from the official Kodi add-on repository. Use the search function in the add-ons section to find `Retrospect` and install it. Detailed info can be found in the [Retrospect wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
 
-**Note:** Make sure the _Auto-Update_ option for Retrospect is enabled to automagically receive new updates.
+**Note:** Make sure the _Auto-Update_ option for Retrospect is turned on to automagically receive new updates.
 
 ![alt text](./resources/media/retrospect01.jpg "The Retrospect information screen")
 
 ### Kodi Krypton    
-If you are running Kodi Krypton (v17) the only way to install Retrospect is to install it from a zip file. Follow these steps (keep in mind they might slightly differ depending on your skin):
-1. Download the latest Retrospect zip file from here: [https://github.com/retrospect-addon/plugin.video.retrospect/releases](https://github.com/retrospect-addon/plugin.video.retrospect/releases)
+If you are running Kodi Krypton (v17) the only way to install Retrospect is to install it from a ZIP file. Follow these steps (keep in mind they might slightly differ depending on your skin):
+1. Download the latest Retrospect ZIP file from here: [https://github.com/retrospect-addon/plugin.video.retrospect/releases](https://github.com/retrospect-addon/plugin.video.retrospect/releases)
 1. Transfer the file to your Kodi system (_optional_).
-1. Use `Install from zip file` in Kodi to install Retrospect.
+1. Use `Install from ZIP file` in Kodi to install Retrospect.
 
-More detailed information can be found in the [Retrospect Wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
+More detailed info can be found in the [Retrospect wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
 
-**Note:** Since Retrospect isn't available in the official Kodi add-on repository for Kodi Krypton (v17), it will not automagically update. Retrospect will notify you of new versions. You will then have to manually update Retrospect using the steps above.
+**Note:** Since Retrospect isn't available in the official Kodi add-on repository for Kodi Krypton (v17), it will not automagically update.
+Retrospect will notify you of new versions. You will then have to manually update Retrospect using the steps above.
 
 # Contributing #
-You can help developing Retrospect via our [Github](https://github.com/retrospect-addon/plugin.video.retrospect) page and/or help translating Retrospect via [Transifex](https://www.transifex.com/retrospect/public/) (A big thanks to Transifex for providing an OS license).
+You can help developing Retrospect via our [GitHub](https://github.com/retrospect-addon/plugin.video.retrospect) page, and/or help translating Retrospect via [Hosted Weblate](https://hosted.weblate.org/projects/retrospect/#components) (A big thanks to Weblate for their services).
```

Comment:
I don't use Weblate (yet), so I don't know why you added it?

Reply:
To smooth out the process of leaving that platform.

Reply:
Ok, but then two things need to happen: Weblate needs to aprove the "Free" version and I would like to move to GPL3 or APGL3 to get rid of the CC license.

Reply:
You can click to ask for approval any time, and changing the license is possible in the Weblate settings.